README.md

```diff
@@ -1,3 +1,7 @@
+# DEPRECATED
+
+This is no longer supported, please consider using https://github.com/0xPolygonHermez/zkevm-contracts/ instead.
```

Comment:
Let's add a date when we will deprecate these contracts.  3 months is the standard so 09/24